src/app/(main)/blog/components/blogPage.tsx

```diff
@@ -3,85 +3,140 @@ import Link from 'next/link';
 import Image from 'next/image';
 import formatDate from '@/lib/formatDate';
 import Post from './post';
+import authors from '../../../data/authors.json';
 
 export default function BlogPage({ posts, blogData, children }: any) {
+	const getAuthors = () => {
+		if (!blogData.authors) return [];
+		const postAuthors = authors.filter(item => blogData.authors.includes(item.id));
+		return postAuthors;
+	};
+
+	const getPrimaryAuthor = () => {
+		if (!blogData.primary_author) return;
+		const primaryAuthor = authors.find(item => item.name === blogData.primary_author?.name);
+		return primaryAuthor;
+	};
+
 	return (
-		<div className="m-auto px-0 pt-200px mobile:pt-150px pb-140px">
-			<div className="max-w-[780px] w-full m-auto px-20px desktop:p-0">
-				{!blogData.isError && (
-					<div className="flex w-full flex-col desktop:flex-row desktop:items-center desktop:justify-between">
-						<div className="font-medium text-12 text-grey-80">
-							<Link href="/blog">Blog</Link>
-
-							<span className="mx-16px">|</span>
-							<span className="text-primary-400 text-12">{blogData.primary_tag}</span>
-						</div>
+		<div className="m-auto px-20px pt-200px mobile:pt-150px pb-140px max-w-[1264px] w-full">
+			<div className="flex items-start tab-min:flex-wrap justify-between">
```

Comment:
This guy here should wrap at the same time the nav bar of the page turns into a mobile view